dub.sdl

```diff
@@ -49,3 +49,105 @@ subPackage {
 
   dependency "dmd:lexer" version="*"
 }
+
+subPackage {
+  name "frontend"
+  targetType "library"
+  stringImportPaths "res"
+  versions "NoBackend"
+  versions "GC"
+  versions "NoMain"
+  sourcePaths
+
+  sourceFiles \
+    "src/ddmd/access.d" \
+    "src/ddmd/aggregate.d" \
+    "src/ddmd/aliasthis.d" \
+    "src/ddmd/apply.d" \
+    "src/ddmd/argtypes.d" \
+    "src/ddmd/arrayop.d" \
+    "src/ddmd/arraytypes.d" \
+    "src/ddmd/astcodegen.d" \
+    "src/ddmd/attrib.d" \
+    "src/ddmd/builtin.d" \
+    "src/ddmd/canthrow.d" \
+    "src/ddmd/clone.d" \
+    "src/ddmd/complex.d" \
+    "src/ddmd/cond.d" \
+    "src/ddmd/constfold.d" \
+    "src/ddmd/cppmangle.d" \
+    "src/ddmd/cppmanglewin.d" \
+    "src/ddmd/ctfeexpr.d" \
+    "src/ddmd/dcast.d" \
+    "src/ddmd/dclass.d" \
+    "src/ddmd/declaration.d" \
+    "src/ddmd/delegatize.d" \
+    "src/ddmd/denum.d" \
+    "src/ddmd/dimport.d" \
+    "src/ddmd/dinifile.d" \
+    "src/ddmd/dinterpret.d" \
+    "src/ddmd/dmacro.d" \
+    "src/ddmd/dmangle.d" \
+    "src/ddmd/dmodule.d" \
+    "src/ddmd/doc.d" \
+    "src/ddmd/dscope.d" \
+    "src/ddmd/dstruct.d" \
+    "src/ddmd/dsymbol.d" \
+    "src/ddmd/dsymbolsem.d" \
+    "src/ddmd/dtemplate.d" \
+    "src/ddmd/dversion.d" \
+    "src/ddmd/escape.d" \
+    "src/ddmd/expression.d" \
+    "src/ddmd/expressionsem.d" \
+    "src/ddmd/frontend.d" \
+    "src/ddmd/func.d" \
+    "src/ddmd/hdrgen.d" \
+    "src/ddmd/id.d" \
+    "src/ddmd/impcnvtab.d" \
+    "src/ddmd/imphint.d" \
+    "src/ddmd/init.d" \
+    "src/ddmd/initsem.d" \
+    "src/ddmd/inline.d" \
+    "src/ddmd/inlinecost.d" \
+    "src/ddmd/intrange.d" \
+    "src/ddmd/json.d" \
+    "src/ddmd/lib.d" \
+    "src/ddmd/link.d" \
+    "src/ddmd/mars.d" \
+    "src/ddmd/mtype.d" \
+    "src/ddmd/nogc.d" \
+    "src/ddmd/nspace.d" \
+    "src/ddmd/objc.d" \
+    "src/ddmd/opover.d" \
+    "src/ddmd/optimize.d" \
+    "src/ddmd/parse.d" \
+    "src/ddmd/sapply.d" \
+    "src/ddmd/templateparamsem.d" \
+    "src/ddmd/semantic.d" \
+    "src/ddmd/sideeffect.d" \
+    "src/ddmd/statement.d" \
+    "src/ddmd/staticassert.d" \
+    "src/ddmd/target.d" \
+    "src/ddmd/typesem.d" \
+    "src/ddmd/traits.d" \
+    "src/ddmd/parsetimevisitor.d" \
+    "src/ddmd/visitor.d" \
+    "src/ddmd/typinf.d" \
+    "src/ddmd/utils.d" \
+    "src/ddmd/statement_rewrite_walker.d" \
+    "src/ddmd/statementsem.d" \
+    "src/ddmd/staticcond.d" \
+    "src/ddmd/safe.d" \
+    "src/ddmd/blockexit.d" \
+    "src/ddmd/printast.d" \
+    "src/ddmd/gluelayer.d"
```

Comment:
Minor detail, it would be nice if the list was sorted alphabetically, unless it's sorted in some other way that I fail to see now.